.saturn/templates.json

```
@@ -1,65 +1,64 @@
 {
   "templates": [
     {
-      "title": "Machine Learning with Snowflake",
-      "thumbnail_image_url": "https://saturn-public-assets.s3.us-east-2.amazonaws.com/example-thumbnails/snowflake.png",
-      "weight": 60,
-      "recipe_path": "examples/snowflake-ml/.saturn/saturn.json"
+      "title": "Dask",
+      "thumbnail_image_url": "https://saturn-public-assets.s3.us-east-2.amazonaws.com/example-thumbnails/dask.png",
+      "weight": 100,
+      "recipe_path": "examples/dask/.saturn/saturn.json"
     },
     {
       "title": "Distributed PyTorch",
       "thumbnail_image_url": "https://saturn-public-assets.s3.us-east-2.amazonaws.com/example-thumbnails/pytorch.png",
-      "weight": 50,
+      "weight": 200,
       "recipe_path": "examples/pytorch/.saturn/saturn.json"
     },
     {
-      "title": "RAPIDS",
-      "thumbnail_image_url": "https://saturn-public-assets.s3.us-east-2.amazonaws.com/example-thumbnails/rapids.png",
-      "weight": 2,
-      "recipe_path": "examples/rapids/.saturn/saturn.json"
+      "title": "RStudio",
+      "thumbnail_image_url": "https://saturn-public-assets.s3.us-east-2.amazonaws.com/example-thumbnails/rstudio.png",
+      "weight": 300,
+      "recipe_path": "examples/api/.saturn/saturn.json"
```

```suggestion
      "recipe_path": "examples/rstudio/.saturn/saturn.json"
```

Just realized this was mapped incorrectly.

---

whoops